### PR TITLE
Initial es_App changes

### PR DESCRIPTION
**Pull request by tpereirasplunk**
Changes to enable ES App tests for SHC and Standalone
Also added tests to check for upgrade scenario with ES App

### CODE REVIEW

File: test/appframework_aws/c3/appframework_aws_test.go [hunk 1]
```diff
@@ -909,26 +909,27 @@ var _ = Describe("c3appfw test", func() {
...
+					SslEnablement: enterpriseApi.SslEnablementIgnore,
 				},
 			}
 
 			// Deploy C3 SVA
-			// Deploy the Cluster Master
-			testcaseEnvInst.Log.Info("Deploy Cluster Manager")
-			_, err = deployment.DeployClusterMaster(ctx, deployment.GetName(), "", "", "")
-			Expect(err).To(Succeed(), "Unable to deploy Cluster Manager")
+			// // Deploy the Cluster Master
```
Comment (jambrosiano) [inline test/appframework_aws/c3/appframework_aws_test.go:923]: Nit: quite a bunch of '// //' in this PR actually

File: test/appframework_aws/c3/appframework_aws_test.go [hunk 2]
```diff
@@ -949,32 +950,76 @@ var _ = Describe("c3appfw test", func() {
...
+			shcPodNames := testenv.GeneratePodNameSlice(testenv.SearchHeadPod, deployment.GetName(), int(shSpec.Replicas), false, 1)
 			deployerPod := []string{fmt.Sprintf(testenv.DeployerPod, deployment.GetName())}
-			shcAppSourceInfo := testenv.AppSourceInfo{CrKind: shc.Kind, CrName: shc.Name, CrAppSourceName: appSourceNameShc, CrAppSourceVolumeName: appSourceVolumeNameShc, CrPod: deployerPod, CrAppVersion: appVersion, CrAppScope: enterpriseApi.ScopeCluster, CrAppList: appListV1, CrAppFileList: appFileList, CrReplicas: int(shc.Spec.Replicas), CrClusterPods: shcPodNames}
+			shcAppSourceInfo := testenv.AppSourceInfo{CrKind: shc.Kind, CrName: shc.Name, CrAppSourceName: appSourceNameShc, CrAppSourceVolumeName: appSourceVolumeNameShc, CrPod: deployerPod, CrAppVersion: appVersion, CrAppScope: enterpriseApi.ScopeLocal, CrAppList: esApp, CrAppFileList: appFileList, CrReplicas: int(shc.Spec.Replicas), CrClusterPods: shcPodNames}
 			allAppSourceInfo := []testenv.AppSourceInfo{shcAppSourceInfo}
 			testenv.AppFrameWorkVerifications(ctx, deployment, testcaseEnvInst, allAppSourceInfo, splunkPodAge, "")
+
+			//############### UPGRADE APPS ################
+			// Delete apps on S3
```
Comment (jambrosiano) [inline test/appframework_aws/c3/appframework_aws_test.go:981]: Nit: May be more clear if changed from 'apps' to 'ES app' or 'ES App V1' , same on other files

File: test/appframework_aws/c3/appframework_aws_test.go (hunk 1 above)
Comment (tpereirasplunk) [reply test/appframework_aws/c3/appframework_aws_test.go:923]: These are comments that should be taken care when we enable indexer Ideally

**Review by jambrosiano — APPROVED** [review]

**Review by akondur — APPROVED** [review]
Looks good :)

**Review by gaurav-splunk — APPROVED** [review]

**Pull request merged**